docs/en/5-templates-syntax.md

```diff
@@ -123,6 +123,29 @@ The template are applied. Result:
 <small class="item item_size_1"></small>
 ```
 
+If second argument of `mod()` was omited or equal is to '*' then templates with any
+value of modifier will be applied.
+
+```js
+block('a').mod('size').tag()('span');
+block('a').mod('size', '*').mix()('sized');
+```
+Both templates will be applied to BEMJSON entities if block equals to 'a' and
```

Comment:
"BEMJSON entities" is a strange thing. Do we declare it somewhere?
[BEM entity](https://en.bem.info/methodology/key-concepts/#bem-entity) is absolutely another thing.